src/Campaigns/Routes/CampaignOverviewStatistics.php

```diff
@@ -0,0 +1,90 @@
+<?php
+
+namespace Give\Campaigns\Routes;
+
+use DateInterval;
+use DatePeriod;
+use DateTimeImmutable;
+use Give\API\RestRoute;
+use Give\Campaigns\CampaignDonationQuery;
+use Give\Campaigns\Models\Campaign;
+use Give\Framework\Support\Facades\DateTime\Temporal;
+
+/**
+ * @unreleased
+ */
+class CampaignOverviewStatistics implements RestRoute
+{
+    /** @var string */
+    protected $endpoint = 'campaign-overview-statistics';
+
+    /**
+     * @unreleased
+     */
+    public function registerRoute()
+    {
+        register_rest_route(
+            'give-api/v2',
+            $this->endpoint,
+            [
+                [
+                    'methods' => 'GET',
+                    'callback' => [$this, 'handleRequest'],
+                    'permission_callback' => function () {
+                        return current_user_can('manage_options');
+                    },
+                ],
+                'args' => [
+                    'campaignId' => [
+                        'type' => 'integer',
+                        'required' => true,
```

Comment:
Should we use `sanitize_callback` here?

Reply:
Resolved, see [20ed797](https://github.com/impress-org/givewp/pull/7545/commits/20ed79783d10264baa636e4f2ae6ba40190b9904).